tests/python_client/testcases/test_utility.py

```diff
@@ -1982,3 +1983,314 @@ def test_delete_user_root(self, host, port):
                                      password=ct.default_password, check_task=ct.CheckTasks.ccr)
         self.utility_wrap.delete_user(user=ct.default_user, check_task=ct.CheckTasks.err_res,
                                       check_items={ct.err_code: 31})
+
+    @pytest.mark.tags(CaseLabel.L3)
+    def test_clear_roles(self, host, port):
+        """
+        target: check get roles list and clear them
```

Comment:
Here may we add the step of "create roles" except for "admin" and "public"? I think only list and drop could not verify the function of drop roles.